UPDATING.md

```diff
@@ -0,0 +1,26 @@
+## Updating this example's dependencies
+
+When you feel like modifying any of this example's dependencies (Mbed Crypto,
+Cryptoauthlib), for example to update the secure element driver interface,
+you'll need to make your changes across a handful of different components.
+
+1. Update SE driver interface (and tests) in Mbed Crypto (`mbed-crypto`)
+1. Save the changes to a branch in your fork of Mbed Crypto (`mbed-crypto`)
+1. Update Mbed OS (`mbed-os`) with your Mbed Crypto fork's SE driver changes
+1. Save the changes to a branch in your fork of Mbed OS (`mbed-os`)
+    1. `cd features/mbedtls/mbed-crypto/importer`
+    1. Edit `CRYPTO_RELEASE` to point to your branch
+    1. Edit `CRYPTO_REPO_URL` to point to your fork
+    1. `make update`; if you have an `mbed-crypto` in a different repo already
```

Comment:
`make update`; if you have **a** `mbed-crypto` in a different repo already

Reply:
No, `an` is correct here. `mbed` starts with a vowel sound, so it uses `an`. This came up once before and we got out the Oxford Guide to English Usage to check :D

Reply:
Yup. In Polish, for example, stuff is spelled exactly as you hear it, so `mbed` would be spelled `embed`, making it obvious that it needs `an` :)